### PR TITLE
Connect from commandline

### PR DESCRIPTION
**Pull request by basicer**
## Short roundup of the initial problem
- When testing network options, it is a pain to get multiple test clients connected

## What will change with this Pull Request?
- Add --connect command line option to connect the client
- Normalize argument parsing

## Screenshots
```
./cockatrice --help
Cockatrice

Options:
  -h, --help                           Displays this help.
  -v, --version                        Displays version information.
  -c, --connect <user:pass@host:port>  Connect on startup
  -d, --debug-output                   Debug to file
```

### CODE REVIEW

**Comment by basicer** [comment]
http://doc.qt.io/qt-5/qcoreapplication.html#applicationVersion-prop

**Comment by tooomm** [comment]
@ebbit1q do you have the same issue with the version command at servatrice?
https://github.com/Cockatrice/Cockatrice/wiki/Servatrice-Command-Line-Reference

Also, are the new commands documented already?

**Comment by ebbit1q** [comment]
No, servatrice always gave the correct version string.

File: cockatrice/src/main.cpp [hunk 1]
```diff
@@ -114,6 +113,21 @@ int main(int argc, char *argv[])
...
+    QCommandLineParser parser;
+    parser.setApplicationDescription("Cockatrice");
+    parser.addHelpOption();
+    parser.addVersionOption();
+
+    parser.addOptions(
+        {{{"c", "connect"}, QCoreApplication::translate("main", "Connect on startup"), "user:pass@host:port"},
+         {{"d", "debug-output"}, QCoreApplication::translate("main", "Debug to file")}});
+
+    parser.process(app);
```
Comment (ZeldaZach) [inline cockatrice/src/main.cpp]: Should we error check to ensure this works?
```cpp
if (!parser.parse(QCoreApplication::arguments())) {
        *errorMessage = parser.errorText();
        return CommandLineError;
    }
```

Comment (basicer) [reply cockatrice/src/main.cpp]: .process() takes care of this, exiting and printing an error message if there is a problem.

**Pull request merged**